docs/API.md

```diff
@@ -0,0 +1,97 @@
+# API
+
+API authentication is based on OAuth2 protocol. You'll have to send username/email and password against a token that will used later.
+Token has 10 minutes of lifetime and refreshToken 30 days of lifetime.
```

Comment:
Why not hours/days of lifetime. Is it a good practice to refresh the access_token regularly ?

Reply:
Apparently this a good practice to refresh the token every 10 minutes.

Reply:
Alright i didn't knows that thx